mqtt3/1.0.0/modules/ROOT/pages/mqtt3-connector-cluster.adoc

```diff
@@ -0,0 +1,10 @@
+= Understand Deploying MQTT3 Connector Apps in a Cluster
+
+When an MQTT broker establishes a connection with a client, the broker identifies the client by its client ID. If the broker establishes a connection with a client whose client ID is `smart-bentley-123`, and then the broker receives a new connection with the same client ID, the broker closes the previous connection (assuming it is stale) and accepts the new connection. This mechanism is known as a client take-over. MQTT brokers employ this mechanism because in certain situations TCP connections can reach a half-open or invalid state, perhaps due to a crash in one of the communication ends, which is undesirable. The broker assumes the new incoming connection is the valid one, closes the older connection, and continues using the newly established connection.
```

Comment:
MQTT broker establishes a connection with a client using its clientID. If the broker establishes a connection with a client whose client ID is `smart-bentley-123`, and then the broker receives a new connection with the same client ID, the broker closes the previous connection (assuming it is stale) and accepts the new connection. This mechanism is known as a client take-over. MQTT brokers employ this mechanism to prevent situations such as TCP connections reaching a half-open or invalid state due to a crash in one of the communication ends, which is undesirable. The broker assumes the new incoming connection to be valid, closes the older connection, and continues using the newly established connection.